### PR TITLE
fix(sso.realm): hardcoded value for realm

### PR DESCRIPTION
**Pull request by corinnekrych**
Fixes https://github.com/fabric8io/fabric8-ui/issues/896

### CODE REVIEW

**Comment by corinnekrych** [comment]
Note this is dependent on https://github.com/fabric8-ui/ngx-login-client/pull/62 to be merged and released

File: config/webpack.dev.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ const HMR = helpers.hasProcessFlag('hot');
 // if env is 'inmemory', the inmemory debug resource is used
 const FABRIC8_FORGE_API_URL = process.env.FABRIC8_FORGE_API_URL || 'https://forge.api.prod-preview.openshift.io';
 const FABRIC8_WIT_API_URL = process.env.FABRIC8_WIT_API_URL || 'https://api.prod-preview.openshift.io/api/';
+const REALM = process.env.REALM || 'fabric8';
```
Comment (joshuawilson) [inline config/webpack.dev.js]: The ENV var should be set in the build config the way "FABRIC8_BRANDING=openshiftio" is on https://github.com/fabric8io/fabric8-ui/blob/master/cico_build_deploy.sh#L27

File: config/webpack.prod.js [hunk 2]
```diff
@@ -33,6 +33,7 @@ const OfflinePlugin = require('offline-plugin');
 const ENV = process.env.NODE_ENV = process.env.ENV = 'production';
 const FABRIC8_FORGE_API_URL = process.env.FABRIC8_FORGE_API_URL;
 const FABRIC8_WIT_API_URL = process.env.FABRIC8_WIT_API_URL;
+const REALM = process.env.REALM || 'fabric8';
```
Comment (joshuawilson) [inline config/webpack.prod.js]: The ENV var should be set in the build config the way "FABRIC8_BRANDING=openshiftio" is on https://github.com/fabric8io/fabric8-ui/blob/master/cico_build_deploy.sh#L27

**Comment by joshuawilson** [comment]
[test]

**Comment by joshuawilson** [comment]
Part of the build is failing with " has no exported member 'REALM'."

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1326-fabric8-ui.int.rdu2c.fabric8.io

**Comment by corinnekrych** [comment]
@joshuawilson I've fixed the merge conflict in release.groovy and now everything is green. ok to merge?

**Pull request merged**